modules/persistence/src/services/metadata/repos_branches/index.ts

```diff
@@ -82,7 +83,7 @@ export const getAllAssociatedRepoBranchesEntries = async (metadata: Metadata) =>
 
   try {
     const db = await pool();
-    const aggregationPipeline = getAggregationPipeline({ project: { $in: fetch } });
+    const aggregationPipeline = getAggregationPipeline({ project: { $in: fetch }, prodDeployable: true });
```

Comment:
AC in the original ticket called for adding this to only `getRepoBranchesEntry`, but it seems like this might be useful for the `getAllAssociatedRepoBranchesEntries` function as well. If there is a project with more than 1 repo (1 deployable, 1 not deployable) in it, both repos could've been returned, but I'm not sure if that's desired. Let me know!

Reply:
Good catch, yes - this is needed!